### ISSUE TITLE
[Feature Request] Support iMessage stickers

### ISSUE THREAD

**Issue by CSab6482** [open-imcore/imcore.react#37]
Below is the same conversation on iOS and in the MyMessage frontend. I zoomed in a bit strangely so that I could fit the entire Instagram DM screenshot in one screenshot (#21). Another thing to note about this screenshot is that the tapbacks are off in the frontend. They are both blue emphasis tapbacks, but they should be gray emphasis tapbacks. Also, my tapback on the "It works" message is a heart and not an emphasis.
![PNG image](https://user-images.githubusercontent.com/53708281/116649976-411e9a80-a94e-11eb-8abb-ef0bdc582367.png)
![no sticker](https://user-images.githubusercontent.com/53708281/116649979-42e85e00-a94e-11eb-83dc-29c647a984a1.png)

**Comment by EricRabil** [comment]
Definitely, the API already supplies sticker info, just no rendering yet.

**Comment by sabogalc** [comment]
Is it safe to close this issue?
